beets/ui/__init__.py

```diff
@@ -1128,6 +1128,11 @@ def _load_plugins(options, config):
     else:
         plugin_list = config['plugins'].as_str_seq()
 
+    # Exclude any plugins that were specified on the command line
+    if options.exclude is not None:
+        plugin_list = [p for p in plugin_list
+                      if p not in options.exclude.split(',')]
```

Comment:
Just making sure: does this work if `-P` is not specified, even without something like `default=''`? I would have expected this to crash because `None` does not have a method `split`…

Reply:
I'm not sure I follow you here. Do you mean if it works when I input `--disable-plugins=` or when I just run a command without `-P` at all like `beet list`?

Reply:
The latter—surely it must work because the tests seem to pass! I'm just a little surprised that the case: i.e., I would have expected leaving `-P` off altogether would crash.

Reply:
It works. I believe it runs only if the option is `not None`. I will post some logs for verification if that's okay?

Reply:
Oh jeez, I totally didn't see the `if options.exclude is not None:` conditional. That would do it!! Sorry for being slow.